tests/test_reco_combined.py

```diff
@@ -50,3 +50,29 @@ def test_accumulate_when_no_results_this_batch(self):
         batch_res, acc_res = metrics.get_score(actual, predicted, batch_accumulate=True, return_extended_results=True)
         self.assertEqual(2, len(batch_res))
         self.assertEqual(2, len(acc_res))
+
+    def test_inter_list_diversity_in_use(self):
```

Comment:
can you add a few tests to test edge cases & expected error behavior of the DiversityRecoMetrics class?

For instance:
- negative `score` for one of the items
-  `num_chunks` less than 1. Some users might be used to `sklearn` API where `-1` means "all"

Reply:
nice edge cases!